### DISCUSSION TITLE
3.2.0

### DISCUSSION THREAD

**Discussion by Saranya-jena** [litmuschaos/litmus#4340] [Releases]
## What's Changed
* Integrate the operator image with scarf gateway by @ispeakc0de in https://github.com/litmuschaos/litmus/pull/4299
* feat: Added view toggle for password input field by @hrishavjha in https://github.com/litmuschaos/litmus/pull/4303
* feat: add tests cases for  chaoscenter web views by @Freedisch in https://github.com/litmuschaos/litmus/pull/4248
* Tests Choascenter: Added test cases for views  by @Freedisch in https://github.com/litmuschaos/litmus/pull/4293
* feat: Test cases for chaosHub views by @Freedisch in https://github.com/litmuschaos/litmus/pull/4249
* Tests: added tests for ChaosCenter web views by @Freedisch in https://github.com/litmuschaos/litmus/pull/4307
* Updated Codeowners by @Saranya-jena in https://github.com/litmuschaos/litmus/pull/4316
* Updated litmus component images with scarf by @amityt in https://github.com/litmuschaos/litmus/pull/4298
* Update Maintainer email by @chandankumar4 in https://github.com/litmuschaos/litmus/pull/4320
* chore: Updates fault docs for 3.0.0 by @neelanjan00 in https://github.com/litmuschaos/litmus/pull/4209
* Feat: Tests cases for Chaoscenter Rest handler in Chaoscenter by @Freedisch in https://github.com/litmuschaos/litmus/pull/4238
* Updated mentoring.md for LFX mentorship by @Saranya-jena in https://github.com/litmuschaos/litmus/pull/4333
* Added installation manifest for 3.2 release by @Saranya-jena in https://github.com/litmuschaos/litmus/pull/4338

**Full Changelog**: https://github.com/litmuschaos/litmus/compare/3.1.0...3.2.0

<hr /><em>This discussion was created from the release <a href='https://github.com/litmuschaos/litmus/releases/tag/3.2.0'>3.2.0</a>.</em>

**Comment by mkos11** [comment]
Great Work
